app/views/categories/show.html.haml

```diff
@@ -1,8 +1,14 @@
-%h1= "#{@category.name}"
+.container
+  %h1.center-align= "#{@category.name}"
```

Comment:
`= "..."` should be rewritten as `...`<br>`%... \#{expression}` can be written without interpolation as `%...= expression`